magicctapipe/utils/functions.py

```diff
@@ -1,294 +1,317 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-import logging
-
 import numpy as np
 import pandas as pd
-import tables
 from astropy import units as u
-from astropy.coordinates import AltAz, EarthLocation, SkyCoord, angular_separation
-from astropy.coordinates.builtin_frames import SkyOffsetFrame
-from astropy.io import fits
-from astropy.table import QTable
-from astropy.time import Time
+from astropy.coordinates import (
+    AltAz,
+    Angle,
+    EarthLocation,
+    SkyCoord,
+    SkyOffsetFrame,
+    angular_separation,
+)
 from ctapipe.coordinates import TelescopeFrame
-from magicctapipe import __version__
-from pyirf.binning import split_bin_lo_hi
-
-logger = logging.getLogger(__name__)
-logger.addHandler(logging.StreamHandler())
-logger.setLevel(logging.INFO)
 
 __all__ = [
     "calculate_disp",
     "calculate_impact",
     "calculate_mean_direction",
-    "calculate_angular_distance",
+    "calculate_off_coordinates",
     "transform_altaz_to_radec",
-    "check_tel_combination",
-    "save_pandas_to_table",
-    "get_dl2_mean",
-    "create_gh_cuts_hdu",
 ]
 
+# The geographic coordinate of ORM
+LON_ORM = -17.89064 * u.deg
```

Comment:
those are also in io/gadf, so maybe better to import them from there

Reply:
Yes, I changed the script as you suggested.

Reply:
For some reasons importing from io/gadf clashes the pipeline, so I changed the code to import them from `magicctapipe.utils.functions`.